src/src/fileLoger/hillel/FileLogger.java

```diff
@@ -0,0 +1,53 @@
+package fileLoger.hillel;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class FileLogger {
+    private FileLoggerConfiguration config;
+
+    public FileLogger(FileLoggerConfiguration config) {
+        this.config = config;
+    }
+
+    public void debug(String message)  {
+        if (config.getLevel().compareTo(LoggingLevel.DEBUG) >= 0) {
+            log("DEBUG", message);
+        }
+    }
+
+    public void info(String message)  {
+        if (config.getLevel().compareTo(LoggingLevel.INFO) >= 0) {
+            log("INFO", message);
+        }
+    }
+
+
+    private void log(String level, String message)  {
+        File file = new File(config.getFile());
+        long fileSize = file.length();
+        long maxFileSize = config.getMaxFileSize();
+        if (fileSize >= maxFileSize) {
+            throw new FileMaxSizeReachedException("Maximum file size reached. " +
+                    "Max size: " + maxFileSize + " bytes. " +
+                    "Current size: " + fileSize + " bytes. " +
+                    "Path: " + config.getFile());
+        }
+        String logEntry = "[" + getCurrentTime() + "][" + level + "] Message: " + message;
```

Comment:
Extract this code `"[" + getCurrentTime() + "][" + level + "] Message: " + message;` to a separate method.
If you would like to concatenate a couple of strings, then it would be better to do it via `StringBuilder`